go/src/koding/klient/machine/mount/mount.go

```diff
@@ -0,0 +1,10 @@
+package mount
+
+// ID is a unique identifier of the mount.
+type ID string
+
+// Mount stores information about a single local to remote machine mount.
+type Mount struct {
+	Path  string `json:"path"`  // Mount point.
+	RPath string `json:"rpath"` // Remote directory path.
```

Comment:
When I see `rpath` I think more about [this](https://en.wikipedia.org/wiki/Rpath) than remote path. What about just `RemotePath string`?